html_text/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Text from HTML field
+====================
+
+This module extracts text from any chunk of HTML, without HTML tags or
```

Comment:
I think it's better to put "This module provides some technical modules that allow to extract ..."